webpack/localization-plugin/README.md

```diff
@@ -0,0 +1,181 @@
+# Localization Plugin for Webpack
+
+## Installation
+
+`npm install @rushstack/localization-plugin --save-dev`
```

Comment:
The LICENSE file still says `@microsoft/localization-plugin`.  Maybe do a full search+replace to be sure.